.gitignore

```diff
@@ -21,3 +21,7 @@ alerts/js/*.min.js
 
 # Generated test data
 tests/data/tmp/*
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```

Comment:
<!-- potential concern, ask for clarification or investigation -->
**❓question:** I know that in CI those are not (yet?) generated, but should we add them preemptively to `.distignore`?

Reply:
YES, thank you! I have some sort of mental block on this!